suite-native/app/app.config.ts

```diff
@@ -61,6 +64,72 @@ if (isCI) {
     }
 }
 
+const getPlugins = (): ExpoPlugins => {
+    let plugins = [
+        [
+            'expo-font',
+            {
+                fonts: [
+                    '../../packages/theme/fonts/TTSatoshi-Medium.otf',
+                    '../../packages/theme/fonts/TTSatoshi-DemiBold.otf',
+                ],
+            },
+        ],
+        [
+            'expo-barcode-scanner',
+            {
+                cameraPermission: 'Allow $(PRODUCT_NAME) to access camera for QR code scanning.',
+            },
+        ],
+        [
+            'expo-build-properties',
+            {
+                android: {
+                    minSdkVersion: 28,
+                },
+                ios: {
+                    deploymentTarget: '14.0',
+                    flipper: true,
+                },
+            },
+        ],
+        '@trezor/react-native-usb/plugins/withUSBDevice.js',
+        // Define FLIPPER_VERSION
+        './plugins/withGradleProperties.js',
+        [
+            '@config-plugins/detox',
+            {
+                subdomains: '*',
+            },
+        ],
+    ];
+
+    // Debug build does not have access to the Sentry token.
+    if (buildType !== 'debug')
+        // EXPLAINER: plugins.push("@sentry...") does not work for some reason during `expo prebuild` and
+        // this plugin is never included in the used array. For this reason the spread operator is used instead.
+        plugins = [
+            ...plugins,
+            [
+                '@sentry/react-native/expo',
+                {
+                    url: 'https://sentry.io/',
+                    authToken: process.env.SENTRY_AUTH_TOKEN,
+                    project: 'suite-native',
+                    organization: 'satoshilabs',
+                },
+            ] as Plugin,
+        ];
+
+    return [
```

Comment:
nitpick (fell free to ignore, it's a matter of taste): I find gradual modification of plugins variable harder to read. I would prefer to keep it as const. You can use this syntax for conditional item in array here:

```javascript
const plugins = [
  ...plugins,
  ...(buildType === 'debug' ? [] : ['@sentry/react-native/expo', {/* url: ... */}]),
  // These should come last
  './plugins/withRemoveXcodeLocalEnv.js',
  // ...
]

```

Reply:
What about simple `plugins.push(...)`?

Reply:
@Nodonisko there is a comment explaining this syntax:

```
// EXPLAINER: plugins.push("@sentry...") does not work for some reason during `expo prebuild` and
// this plugin is never included in the used array. For this reason the spread operator is used instead.
```

Reply:
@matejkriz good idea, fixed by 5d9543b375